build_tools/ci/run_matmul_test.sh

```diff
@@ -131,15 +136,41 @@ function run_matmul_test() {
       --acc_type=${acc_type} \
       --shapes=${shapes}
 
-  echo "**** Generating .vmfb files ****"
-  ${IREE_COMPILE_EXE} \
-      "${OUTPUT_DIR}/${name}_matmuls.mlir" \
-      --iree-hal-target-backends=${target_backend} \
-      --iree-amd-aie-peano-install-dir=${peano_install_path} \
-      --iree-amd-aie-mlir-aie-install-dir=${mlir_aie_install_path} \
-      --iree-amd-aie-vitis-install-dir=${vitis_path} \
-      --iree-hal-dump-executable-files-to=$PWD \
-      -o "${OUTPUT_DIR}/${name}_matmuls.vmfb"
+  echo "**** Generating .vmfb files for $pipeline pipeline ****"
```

Comment:
can we just do --iree-amdaie-use-pipeline=$pipeline? and let iree-compile fail if its not a valid pipeline instead of handling that here?

Reply:
Sure - good point. I'll do that.